src/main/java/com/wultra/security/userdatastore/userclaims/UserClaimsEntity.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * User Data Store
+ * Copyright (C) 2023 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.security.userdatastore.userclaims;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+/**
+ * Entity for user claims.
+ *
+ * @author Lubos Racansky lubos.racansky@wultra.com
+ */
+@Entity
+@Table(name = "ud_user_claims")
+@Getter
+@Setter
+class UserClaimsEntity {
+
+    @Id
+    private String userId;
+
+    private String claims;
+
+    private LocalDateTime timestampCreated = LocalDateTime.now();
+
+    private LocalDateTime timestampUpdated;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
I think there are better ways to handle the class comparison. This comparison does not e.g. take into account proxies.

Reply:
I will regenerate it with the option _accept subclasses_.